parser/src/parse.ts

```diff
@@ -241,7 +250,7 @@ export async function parse(rawArgs: string[]) {
       }),
     "Writing data to file",
     `Wrote data to ${resolve(outPath)}`,
```

Comment:
We already use resolve as a named import here

Reply:
Oh then probably I should remove the extra import